README.md

```diff
@@ -18,14 +18,14 @@ graph LR
   M--pickup-->D((Reciever))
 ```
 
- - **CI** automate builds and tests all push to the main branch also as all PRs created.
+ - **CI** automates builds and tests all pushes to the main branch as well as all PRs created.
```

Comment:
- **CI** automates builds and tests that push to the main branch for all PRs created.